dev/system-test/firestore.ts

```diff
@@ -43,12 +43,14 @@ import {
   postConverterMerge,
   verifyInstance,
 } from '../test/util/helpers';
-import IBundleElement = firestore.IBundleElement;
```

Comment:
Thanks for re-ordering. Can you remove the empty line?

Reply:
Done.